spec/services/make_an_offer_spec.rb

```diff
@@ -1,14 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe MakeAnOffer do
+  let(:auth) { Authorisation.new(actor: SupportUser.new) }
```

Comment:
The policy of "SupportUsers can do everything" is accidentally part of this test. What would you think of something like

`double(assert_can_make_offer!: true)`

?

Reply:
Yes, something along these lines would be good.